### PR TITLE
ParallelsInternationalGmbH.ParallelsClient-64bit version 18.0.22546

### PR DESCRIPTION
**Pull request by ghost**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/18009)

### CODE REVIEW

File: manifests/p/ParallelsInternationalGmbH/ParallelsClient-64bit/18.0.22546/ParallelsInternationalGmbH.ParallelsClient-64bit.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: ParallelsInternationalGmbH.ParallelsClient-64bit
+PackageVersion: 18.0.22546
+PackageLocale: en-US
+Publisher: Parallels International GmbH
+PackageName: Parallels Client-64 bit
```
Comment (ItzLevvie) [inline manifests/p/ParallelsInternationalGmbH/ParallelsClient-64bit/18.0.22546/ParallelsInternationalGmbH.ParallelsClient-64bit.locale.en-US.yaml:8]:
```suggestion
PackageName: Parallels Client
```

File: manifests/p/ParallelsInternationalGmbH/ParallelsClient-64bit/18.0.22546/ParallelsInternationalGmbH.ParallelsClient-64bit.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: ParallelsInternationalGmbH.ParallelsClient-64bit
```
Comment (ItzLevvie) [inline manifests/p/ParallelsInternationalGmbH/ParallelsClient-64bit/18.0.22546/ParallelsInternationalGmbH.ParallelsClient-64bit.yaml:4]:
```suggestion
PackageIdentifier: Parallels.Parallels
```
**Please make sure to update the folder names and file names using GitHub Desktop from https://desktop.github.com on Windows.**

File: manifests/p/ParallelsInternationalGmbH/ParallelsClient-64bit/18.0.22546/ParallelsInternationalGmbH.ParallelsClient-64bit.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: ParallelsInternationalGmbH.ParallelsClient-64bit
```
Comment (ItzLevvie) [inline manifests/p/ParallelsInternationalGmbH/ParallelsClient-64bit/18.0.22546/ParallelsInternationalGmbH.ParallelsClient-64bit.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: Parallels.Parallels
```
**Please make sure to update the folder names and file names using GitHub Desktop from https://desktop.github.com on Windows.**

File: manifests/p/ParallelsInternationalGmbH/ParallelsClient-64bit/18.0.22546/ParallelsInternationalGmbH.ParallelsClient-64bit.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: ParallelsInternationalGmbH.ParallelsClient-64bit
```
Comment (ItzLevvie) [inline manifests/p/ParallelsInternationalGmbH/ParallelsClient-64bit/18.0.22546/ParallelsInternationalGmbH.ParallelsClient-64bit.installer.yaml:4]:
```suggestion
PackageIdentifier: Parallels.Parallels
```
**Please make sure to update the folder names and file names using GitHub Desktop from https://desktop.github.com on Windows.**

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Please read my other reviews.

**Comment by ItzLevvie** [comment]
cc @RadSpaghetti: Please make sure to update the folder names and file names using GitHub Desktop from https://desktop.github.com on Windows.

**Comment by jedieaston** [comment]
I'm getting errors when attempting to install this:

![image](https://user-images.githubusercontent.com/21368066/122920672-a29c2d80-d32f-11eb-8242-6eae3a11023e.png)

```
C:\Users\WDAGUtilityAccount\Desktop\SandboxTest\18.0.22546\                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              ██████████████████████████████  33.9 MB / 33.9 MB
Successfully verified installer hash
Starting package install...
Installer failed with exit code: 1603
Installer log is available at: C:\Users\WDAGUtilityAccount\AppData\Local\Packages\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\LocalState\DiagOutputDir\WinGet-Parallels.Parallels.18.0.22546-2021-06-22-07-57-46.859.log
--> Refreshing environment variables
--> Getting list of installed applications...
Name                  Id                                            Version          Available   Source
-------------------------------------------------------------------------------------------------------
Microsoft Edge        Microsoft.Edge                                90.0.818.56      91.0.864.54 winget
Microsoft Edge Update Microsoft Edge Update                         1.3.143.57
App Installer         Microsoft.DesktopAppInstaller_8wekyb3d8bbwe   1.11.11451.0
Microsoft Edge        Microsoft.MicrosoftEdge.Stable_8wekyb3d8bbwe  90.0.818.56
Windows Security      Microsoft.SecHealthUI_8wekyb3d8bbwe           1000.21390.1.0
WinGet MSStore Source Microsoft.Winget.MSStore.Source_8wekyb3d8bbwe 2021.328.455.669
WinGet Source         Microsoft.Winget.Source_8wekyb3d8bbwe         2021.622.837.717

PS C:\Users\WDAGUtilityAccount\Desktop\Parallels>
```

Can another moderator test this and make sure there's not some conflict with running it in the sandbox?

@ItzLevvie
@OfficialEsco

**Review by ItzLevvie — APPROVED** [review]
image

**Pull request merged**